_posts/2024-7-15-Using Managed Identity DaaS storage accounts.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Managed Identity Support for Storage Account used in Diagnostics under Diagnose and Solve blade"
+author_name: "Puneet Gupta"
+category: 'Diagnostics'
+---
+
+We are pleased to announce **Managed Identity** support for authenticating against storage accounts used for diagnostic tools under Diagnostics and a Service. This feature allows the use of both system-assigned and user-assigned managed identities as authentication mechanisms for connecting to storage accounts where memory dumps and other diagnostic data are stored.
+
+### Key Benefits
+- Enhanced Security: Managed Identity enhances security by eliminating the need for Key-based authentication on the storage account.
+- Compatibility: Managed Identity for storage accounts can be used with applications hosted on both Windows and Linux App Service plans.
+
+### Usage Details
```

Comment:
"Just a suggestion, I think it would be nice to have the instructions on how to access the configuration page with a numbered list instead."

Ex)

1. Navigate to your App Service web app in the Azure Portal.
2. Select **Diagnose and Solve Problems**.
3. Select **Diagnostics Tools**. 
...